lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,13 @@ class InteractionHandler {
 
   onContext(event) {
     let pointer = this.getPointer({x:event.clientX, y:event.clientY});
-    this.checkSelectionChanges(pointer, event);
-    this.selectionHandler._generateClickEvent('oncontext', event, pointer);
+    const hoveredElements = {
+      nodes: (this.selectionHandler.getNodeAt(pointer) || []),
```

Comment:
You are using the entire node and edge object here. What should be set just just the node and edge id's (conforming to documentation).